src/graphql/account.rs

```diff
@@ -641,6 +742,7 @@ fn initial_credential() -> anyhow::Result<types::Account> {
         None,
         None,
     )?;
+    initial_account.update_last_signin_time();
```

Comment:
Could you explain why you put this?

Reply:
This is the part set to satisfy the existing test case.
There is also a part where admin/admin does not affect the part you are currently using when running for the first time.
In the future, I thought of the expected behavior of creating a new account after logging in with the admin account, and requesting password change when attempting to log in with the created account.

Reply:
I don't quite understand. Could you elaborate on what you answered?

Reply:
This `initial_credential` seems to create the first admin account and immediately set its sign-in time. This makes it pointless to alter `sign_in` to check the last sign-in time. What do you think?

Reply:
This part has been restored to its original state.